addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -791,7 +820,7 @@ blockquote {
 .btn-custom:hover, .btn-fill-custom:hover {
     filter: invert(0.2);
 }
-.btn-outline-custom {
+.btn-outline-custom, .btn-outline-primary, .btn-outline-secondary {
```

Comment:
Mmh, I don't understand this change (why not extend above like for fill for instance?) but I in fact does not understand the CSS rules here, maybe it is worth fixing in stable:
```
background-color: '';
```
is not a valid CSS property :grimacing:

Reply:
Created #86205 to use valid CSS properties in 15.0.